cmd/utils/flags.go

```diff
@@ -157,6 +157,10 @@ var (
 		Name:  "fast",
 		Usage: "Enables fast syncing through state downloads",
 	}
+	LightKDFFlag = cli.BoolFlag{
+		Name:  "lightkdf",
+		Usage: "Reduce KDF iterations trading memory usage vs security",
```

Comment:
Can be improved a bit with "Reduce KDF memory usage at some expense of KDF strength"